flowzone.yml

```
@@ -469,8 +469,16 @@
     with:
       version: v0.14.0
 
+  - &setupPython # https://github.com/actions/setup-python
+    name: Setup python
+    id: setup-python
+    uses: actions/setup-python@61a6322f88396a6271a6ee3565807d608ecaddd1 # v4
```

Why not 
```suggestion
    uses: actions/setup-python@v4
```

I see specific commits referenced throughout the config, is it our convention to minimize the risk of breaking things?

---

Yes, Renovate will manage these digests automatically for us but the goal is to reduce the risk of introducing changes without a commit record.